types/openlayers/index.d.ts

```diff
@@ -144,18 +121,14 @@ declare module ol {
         /**
          * Add elements to the collection.  This pushes each item in the provided array
          * to the end of the collection.
-         * @param {!Array.<T>} arr Array.
-         * @return {ol.Collection.<T>} This collection.
          * @api stable
          */
         extend(arr: T[]): ol.Collection<T>;
 
         /**
          * Iterate over each element, calling the provided callback.
-         * @param {function(this: S, T, number, Array.<T>): *} f The function to call
```

Comment:
Need to be careful not to leave dangling doc when tags are removed.

Reply:
Oups, I guess my regex was a little bit too aggressive :flushed: